randovania/game_description/game_migration.py

```diff
@@ -299,6 +299,23 @@ def _migrate_v23(data: dict) -> dict:
     return data
 
 
+def _migrate_v24(data: dict) -> dict:
+    regions_data = data["regions"]
+
+    # find all dock nodes
+    all_nodes = [
+        node
+        for region in regions_data
+        for area_name, area in region["areas"].items()
+        for node_name, node in area["nodes"].items()
+        if node["node_type"] == "dock"
+    ]
+    for node in all_nodes:
+        node["ui_custom_name"] = None
```

Comment:
Will this not cause issues for games that actually *do* define this now? i.e impact crater in prime1

Reply:
Not really but you are right that this isn't a good way.
It isn't an issue with the databases because I migrated them and then edited the custom names, like you can see in the json files. 
I should take the current state of the databases in the PR as blue print and add the ui_custom_names in the migration files with `region/area/node => custom name`, so the migration would create exactly the state of how it is in the PR.

Also: Is the custom name something for the pretty print iff != null/None?

Reply:
Oh I didn't even think of that.
Yeah would be nice to have it in the pretty print.

Reply:
I rebased and moved the stuff to the migration file. Like before: First commit is relevant stuff, second commit is only `database refresh-game-description`